linera-client/src/client_context.rs

```diff
@@ -640,12 +640,24 @@ where
         vm_runtime: VmRuntime,
     ) -> Result<ModuleId, Error> {
         info!("Loading bytecode files");
-        let contract_bytecode = Bytecode::load_from_file(&contract)
-            .await
-            .with_context(|| format!("failed to load contract bytecode from {:?}", &contract))?;
-        let service_bytecode = Bytecode::load_from_file(&service)
-            .await
-            .with_context(|| format!("failed to load service bytecode from {:?}", &service))?;
+        let contract_bytecode = Bytecode::load_from_file(&contract).await.map_err(|e| {
```

Comment:
(Would be great to do this inside `load_from_file` directly. But that would probably mean we'd have yet another error type?)

Reply:
That method returns `io::Error` and I didn't want to change that to the custom `Inner` error type (which has some macro magic enriched that could make it more complicated than necessary).